app/components/results-list/component.js

```diff
@@ -47,30 +47,18 @@ export default Ember.Component.extend({
         },
 
         pageback() {
-            let queryParams = {};
             let page = Number(this.parameters["page"]);
-            if (!page) {
-                page = 0;
-            }
-            page = page-10;
-            if (page < 0) {
-                page = 0;
+            if (!page || --page < 1) {
+                page = 1;
             }
-            queryParams["page"] = page
-            this.attrs.transitionToFacet("search", queryParams);
+            this.attrs.transitionToFacet("search", {page: page})
         },
         pagenext() {
-            let queryParams = {};
             let page = Number(this.parameters["page"]);
-            if (!page) {
-                page = 0;
-            }
-            page = page+10;
-            if (page < 0) {
-                page = 0;
+            if (!page || ++page < 1) {
```

Comment:
if pagenum isn't defined, or after incrementing page by one, the page is still less than one, default to page = 1